### DISCUSSION TITLE
Qlora 微调模型测试

### DISCUSSION THREAD

**Discussion by TousenKaname** [open-compass/opencompass#872] [Q&A]
我现在有一个基于 internlm2 的 qlora 微调的模型，最终结果是这样的一个目录结构
<img width="403" alt="image" src="https://github.com/open-compass/opencompass/assets/55678087/2841f362-b42b-45db-be91-82038b786dc8">
我想测试 epoch3.pth文件夹里面的pt 的 tokenizer，请问我该怎么写 model

**Comment by onlyone2019** [comment]
请问这个问题您解决了吗？我也苦于这个问题

**Reply by bittersweet1999** [reply]
Is this model trained with Xtuner? If so, you would need to first use the relevant scripts to convert it into Hugging Face format checkpoints for testing. You can find the relevant scripts within Xtuner.
